### ISSUE TITLE
Support for Explainable Boosting Machine

### ISSUE THREAD

**Issue by KalinNonchev** [konstantint/SKompiler#13]
Hello,

thank you for the package. Would you also offer support for Explainable Boosting Machine from [InterpretML](https://interpret.ml/)?
It is similar to Gradient Boosting Machine of sklearn.

Thank you in advance!

Best,

**Comment by konstantint** [comment]
I don't believe I would personally find time to implement that very soon. I am not familiar with the EBM model currently but looking into [implementation](https://github.com/interpretml/interpret/blob/38a4176004e0233dfdd9a8afc1a5e671357cbb18/python/interpret-core/interpret/glassbox/ebm/bin.py#L2128) I can't immediately tell how complex it is to translate it into a SK expression (from a quick glance it seems the logic is more complicated than what one would expect from a straightforward boosting model).

**Comment by paulbkoch** [comment]
Our docs have a simplified replica of the predict function that might be useful.

https://interpret.ml/docs/ebm-internals-regression.html
